modules/custom/az_paragraphs/css/az_paragraphs_full_width.css

```
@@ -1,10 +1,18 @@
 .nav-link, .nav-pills .nav-link, .nav-tabs .nav-link {
   background-color: white;
 }
+.full-width-background {
+    max-width: calc(100vw - var(--scrollbar-width) / 2);
+}
+
+.layout-sidebar-first .full-width-background,
+.layout-sidebar-second .full-width-background {
+    max-width: calc(100vw - var(--scrollbar-width));
```

```suggestion
    max-width: calc(100vw - var(--scrollbar-width)/2);
```
This fixes it for me.